ui/apps/platform/src/Containers/Collections/CollectionResults.tsx

```diff
@@ -12,6 +12,8 @@ import {
     SelectOption,
     debounce,
     Title,
+    Skeleton,
+    Spinner,
```

Comment:
Small request: because the names are almost in order, can you move the last few into their places?